doc/contributing/hw/comportability/README.md

```diff
@@ -356,8 +356,8 @@ In this example, the IP name is `uart`, though the other configuration fields ar
     ],
     available_inout_list: [],        // optional; default []
     interrupt_list: [                // optional; default []
-      { name: "tx_watermark",  desc: "raised if the transmit FIFO..."}
-      { name: "rx_watermark",  desc: "raised if the receive FIFO..."}
+      { name: "tx_watermark",  desc: "raised if the transmit FIFO...", type: "status"}
+      { name: "rx_watermark",  desc: "raised if the receive FIFO...", type: "status"}
```

Comment:
Do we need to add text somewhere that mentions that all the other interrupts are of event type?

Reply:
The table under (### Defining Interrupts) mentions this default. However, I added a comment in the example .hjson as well.